build-test.cmd

```diff
@@ -360,6 +366,20 @@ if errorlevel 1 (
     exit /b 1
 )
 
+xcopy /s /y "%CORE_ROOT_STAGE%" "%CORE_ROOT%"
```

Comment:
No, each job would attempt to crossgen the same Core_Root. This would happen N times, but the operation is quick, and doesn't create duplicates. @MattGal would this introduce an unwanted Race condition?